_data/merchants.yml

```diff
@@ -140,6 +140,8 @@
       url: http://www.guitartheoryrevolution.info/blog/guitar-theory-revolution-store/
     - name: Infield Loan Services - Atlanta, Construction Consulting, Contract review, Feasibility, Funds Escrow
       url: mailto:info@loandraw.com
+	- name: KoDDoS.net - DDoS Protection
```

Comment:
Looks like this is breaking the formatting (I think you used tab instead of spaces)

Reply:
Sorry, I missed that. Fixed.